stake-pool/py/tests/test_bot_rebalance.py

```diff
@@ -19,7 +19,10 @@ async def test_rebalance_this_is_very_slow(async_client, validators, payer, stak
     (stake_pool_address, validator_list_address) = stake_pool_addresses
     resp = await async_client.get_minimum_balance_for_rent_exemption(STAKE_LEN)
     stake_rent_exemption = resp['result']
-    increase_amount = MINIMUM_ACTIVE_STAKE
+    # With minimum delegation at MINIMUM_DELEGATION + rent-exemption, when
+    # decreasing, we'll need rent exemption + minimum delegation delegated to
+    # cover all movements
+    increase_amount = MINIMUM_ACTIVE_STAKE + stake_rent_exemption
```

Comment:
ohhh it was the 1.11 change that everything needs to be rent exempt, i understand now